failure in TestAllocatorFuzz unit test

I got this once while working on #914, though that PR does not change anything significant in IPAM.

````
INFO: 2015/06/14 10:46:04.349391 [allocator 01:00:00:01:00:00] Shutdown
panic: Dup found for address 10.0.1.210 - second4279 and second2257

goroutine 74 [running]:
github.com/weaveworks/weave/ipam.func·029(0xc208fd68e0, 0xa)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:356 +0x4bf
github.com/weaveworks/weave/ipam.func·035(0x10b7)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:453 +0x1c5
github.com/weaveworks/weave/ipam.func·032(0x8)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 1 [chan receive]:
testing.RunTests(0xa387a8, 0xddbf60, 0xc, 0xc, 0xa3bb01)
	/usr/local/go/src/testing/testing.go:556 +0xad6
testing.(*M).Run(0xc20802e8c0, 0x1329720)
	/usr/local/go/src/testing/testing.go:485 +0x6c
main.main()
	github.com/weaveworks/weave/ipam/_test/_testmain.go:128 +0x291

goroutine 5 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/local/go/src/os/signal/signal_unix.go:27 +0x35

goroutine 16 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 18 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc209115200, 0xc20914a8a0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 24 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 17 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 19 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc2091152c0, 0xc20914a900)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 31 [chan receive]:
github.com/weaveworks/weave/ipam.(*Allocator).Encode(0xc2091ef680, 0x0, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:458 +0x1a3
github.com/weaveworks/weave/ipam.(*ipamGossipData).Encode(0xc20934a578, 0x0, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:484 +0x54
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:261 +0x3b1
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 37 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc2091efec0, 0xc20935a4e0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 38 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 33 [chan receive]:
github.com/weaveworks/weave/ipam.(*Allocator).Encode(0xc2091ef680, 0x0, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:458 +0x1a3
github.com/weaveworks/weave/ipam.(*ipamGossipData).Encode(0xc20934a578, 0x0, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:484 +0x54
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:261 +0x3b1
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 35 [semacquire]:
sync.(*WaitGroup).Wait(0xc209c5f7c0)
	/usr/local/go/src/sync/waitgroup.go:132 +0x169
github.com/weaveworks/weave/ipam.func·033(0x1388, 0xc209c5f7a0)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:435 +0x1aa
github.com/weaveworks/weave/ipam.TestAllocatorFuzz(0xc20807cbd0)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:463 +0x72a
testing.tRunner(0xc20807cbd0, 0xddc008)
	/usr/local/go/src/testing/testing.go:447 +0xbf
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:555 +0xa8b

goroutine 36 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 39 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a000, 0xc20935a5a0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 40 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 41 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a0c0, 0xc20935a660)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 42 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 43 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a180, 0xc20935a720)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 44 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 45 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a240, 0xc20935a7e0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 46 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 47 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a300, 0xc20935a8a0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 48 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 49 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a3c0, 0xc20935a960)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 50 [chan receive]:
github.com/weaveworks/weave/ipam.(*Allocator).OnGossipUnicast(0xc20936a480, 0x10000020000, 0xc2095d09c0, 0x37, 0x40, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:402 +0x4bd
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:257 +0x1d0
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 51 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a480, 0xc20935aa20)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 52 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 53 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a540, 0xc20935aae0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 54 [select]:
github.com/weaveworks/weave/ipam.func·042()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:245 +0x70a
created by github.com/weaveworks/weave/ipam.(*TestGossipRouter).connect
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/testutils_test.go:271 +0x1bc

goroutine 55 [select]:
github.com/weaveworks/weave/ipam.(*Allocator).actorLoop(0xc20936a600, 0xc20935aba0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:508 +0x1cf
created by github.com/weaveworks/weave/ipam.(*Allocator).Start
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:70 +0x87

goroutine 68 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).Allocate(0xc20936a180, 0xc208fd6c60, 0xa, 0xa0003ff0a000001, 0x0, 0xa, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:184 +0x1cf
github.com/weaveworks/weave/ipam.func·029(0xc208fd6c60, 0xa)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:342 +0x136
github.com/weaveworks/weave/ipam.func·035(0x507)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:453 +0x1c5
github.com/weaveworks/weave/ipam.func·032(0x2)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 70 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).Free(0xc20936a480, 0xc2090a16e8, 0x8, 0xa00025d, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:283 +0x22d
github.com/weaveworks/weave/ipam.func·030()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:391 +0x321
github.com/weaveworks/weave/ipam.func·035(0x8ea)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:457 +0xac
github.com/weaveworks/weave/ipam.func·032(0x4)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 71 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).Free(0xc20936a300, 0xc2090dbbb0, 0xa, 0xa0001ee, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:283 +0x22d
github.com/weaveworks/weave/ipam.func·030()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:391 +0x321
github.com/weaveworks/weave/ipam.func·035(0xadd)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:457 +0xac
github.com/weaveworks/weave/ipam.func·032(0x5)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 66 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).Free(0xc2091efec0, 0xc208964de8, 0x8, 0xa00002d, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:283 +0x22d
github.com/weaveworks/weave/ipam.func·030()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:391 +0x321
github.com/weaveworks/weave/ipam.func·035(0x120)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:457 +0xac
github.com/weaveworks/weave/ipam.func·032(0x0)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 69 [chan receive]:
github.com/weaveworks/weave/ipam.(*Allocator).Allocate(0xc20936a180, 0xc208fce660, 0xa, 0xa0003ff0a000001, 0x0, 0xa, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:184 +0x1cf
github.com/weaveworks/weave/ipam.func·029(0xc208fce660, 0xa)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:342 +0x136
github.com/weaveworks/weave/ipam.func·035(0x6fe)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:453 +0x1c5
github.com/weaveworks/weave/ipam.func·032(0x3)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 72 [chan receive]:
github.com/weaveworks/weave/ipam.(*Allocator).Allocate(0xc20936a000, 0xc208fd67a0, 0xa, 0xa0003ff0a000001, 0x0, 0xa, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:184 +0x1cf
github.com/weaveworks/weave/ipam.func·029(0xc208fd67a0, 0xa)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:342 +0x136
github.com/weaveworks/weave/ipam.func·035(0xce2)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:453 +0x1c5
github.com/weaveworks/weave/ipam.func·032(0x6)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 73 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).Allocate(0xc2091efec0, 0xc208fd6ae0, 0xa, 0xa0003ff0a000001, 0x0, 0xa, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:184 +0x1cf
github.com/weaveworks/weave/ipam.func·029(0xc208fd6ae0, 0xa)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:342 +0x136
github.com/weaveworks/weave/ipam.func·035(0xedb)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:453 +0x1c5
github.com/weaveworks/weave/ipam.func·032(0x7)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 75 [chan receive]:
github.com/weaveworks/weave/ipam.(*Allocator).Allocate(0xc20936a0c0, 0xc208fce5a0, 0xa, 0xa0003ff0a000001, 0x0, 0xa, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:184 +0x1cf
github.com/weaveworks/weave/ipam.func·029(0xc208fce5a0, 0xa)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:342 +0x136
github.com/weaveworks/weave/ipam.func·035(0x12b6)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:453 +0x1c5
github.com/weaveworks/weave/ipam.func·032(0x9)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183

goroutine 67 [runnable]:
github.com/weaveworks/weave/ipam.(*Allocator).Allocate(0xc20936a0c0, 0xc208f98460, 0xa, 0xa0003ff0a000001, 0x0, 0xc20a000297, 0x0, 0x0)
	github.com/weaveworks/weave/ipam/_test/_obj_test/allocator.go:184 +0x1cf
github.com/weaveworks/weave/ipam.func·031()
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:408 +0x23d
github.com/weaveworks/weave/ipam.func·035(0x30b)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:461 +0x90
github.com/weaveworks/weave/ipam.func·032(0x1)
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:431 +0x7e
created by github.com/weaveworks/weave/ipam.func·033
	/home/matthias/go/src/github.com/weaveworks/weave/ipam/allocator_test.go:433 +0x183
FAIL	github.com/weaveworks/weave/ipam	5.665s
````

--- a/ipam/allocator_test.go
+++ b/ipam/allocator_test.go
@@ -289,10 +289,10 @@ func TestAllocatorFuzz(t *testing.T) {
 	common.InitDefaultLogging(false)
 	const (
 		firstpass    = 1000
-		secondpass   = 5000
+		secondpass   = 20000
 		nodes        = 10
 		maxAddresses = 1000
-		concurrency  = 10
+		concurrency  = 30
 		cidr         = "10.0.1.7/22"
 	)
 	allocs, _, subnet := makeNetworkOfAllocators(nodes, cidr)
@@ -398,12 +398,16 @@ func TestAllocatorFuzz(t *testing.T) {
 		addrIndex := rand.Int31n(int32(len(addrs)))
 		addr := addrs[addrIndex]
 		res := state[addr]
+		if res.block {
+			stateLock.Unlock()
+			return
+		}
 		res.block = true
 		state[addr] = res
 		stateLock.Unlock()
 		alloc := allocs[res.alloc]
 
-		//common.Info.Printf("Asking for %s on allocator %d again", addr, res.alloc)
+		//common.Info.Printf("Asking for %s (%s) on allocator %d again", res.name, addr, res.alloc)
 
 		newAddr, _ := alloc.Allocate(res.name, subnet, nil)
 		oldAddr, _ := address.ParseIP(addr)

